### DISCUSSION TITLE
Performance Issue

### DISCUSSION THREAD

**Discussion by gl0zzy** [DistroAV/DistroAV#966] [Q&A - Support]
I have two PC devices, Device A and Device B. Device A has better specifications and serves as a gaming machine, while Device B has average specifications and functions as a streaming machine. Device A uses the Intel(R) Wireless-AC 9560 160MHz network card, and Device B uses the Intel(R) WIFI 6 AX200 network card.
I am currently facing an issue where, when I use Device A to output the screen via NDI to Device B, the display is very laggy. However, when I use Device B to output the screen via NDI to Device A, the display is smooth. How can I resolve this issue?

**Comment by paulpv** [comment]
First recommendation is to not use wireless.
Seriously.

What OS?

Is this using obs-ndi?

Have you tried just a pure NDI Screen Capture plus NDI Studio Monitor?

**Reply by gl0zzy** [reply]
There is no way to use a wired connection in my environment.
Both Windows 10 22h2
Yes, I use obs-ndi.
I don't know much about ndi and am just using it using obs-ndi.

**Reply by paulpv** [reply]
Install https://ndi.video/tools/ and use their `NDI Screen Capture` and `NDI Studio Monitor` to test with.
If that pure NDI official solution isn't working well then obs-ndi never will work better.
Get that to work well and obs-ndi might start working.

There is always a way to get a wired connection to work! :)

**Comment by paulpv** [comment]
BTW, I don't mind conversing here, but this might be easier to discuss on our Discord: https://discord.gg/ZuTxbUK3ug
